app/jobs/person/inactivity_block_warning_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Pfadibewegung Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class Person::InactivityBlockWarningJob < RecurringJob
+  def perform
+    return unless Person::BlockService.warn?
+
+    Person::BlockService.warn_scope.find_each do |person|
```

Comment:
Ich denke es würde Sinn machen, die Logik direkt in den `BlockService` zu verschieben, z.B. so ähnlich:

```
class Person::BlockService
  def self.send_warnings
    return unless warn?
    
    warn_scope.find_each {|p| self.new(p).inactivity_warning! }
  end
end

class Person::InactivityBlockWarningJob
  def perform
    Person::BlockService.send_warnings
    true
  end
end
```